it_IT/readme.md

```diff
@@ -0,0 +1,951 @@
+
+Mocha richiede che tu usi funzioni globali come `describe` o `it` con l'interfaccia di default (usata dalla maggior parte delle persone). Non impone una struttura precisa ed esegue test in serie senza alcun isolamento a livello di processo, rendendolo lento.
+
+Tape e tap sono buoni moduli. AVA ha preso ispirazione dalla loro sintassi. Anche loro eseguono test in serie. Il loro output [TAP](https://testanything.org) integrato non è molto pratico da usare e quindi si finisce sempre per usare un reporter TAP esterno.
+
+Viceversa AVA impone una struttura precisa ed esegue i test in modo concorrente, ogni file di test in un processo separato. Il suo reporter integrato è facile da comprendere a prima vista, nonostante ciò AVA supporta anche l'output TAP mediante il parametro da CLI.
+
+### Come posso utilizzare reporter personalizzati?
+
+AVA supporta il formato TAP ed è quindi compatibile con ogni [TAP reporter](https://github.com/sindresorhus/awesome-tap#reporters). Usa il [parametro `--tap`](#produzione-tap-opzionale) per abilitare l'output TAP.
+
+### Come si scrive e pronuncia AVA?
+
+AVA, no Ava o ava. Si pronuncia [`/ˈeɪvə/` ay-və](https://github.com/sindresorhus/ava/blob/master/media/pronunciation.m4a?raw=true).
+
+### Cos'è lo sfondo del logo in cima?
+
+La [galassia di Andromeda](https://es.wikipedia.org/wiki/Galaxia_de_Andr%C3%B3meda).
+
+### Qual è la differenza tra concorrenza e parallelismo?
+
+[Concorrenza non è parallelismo, ma lo abilita.](http://stackoverflow.com/q/1050222)
+
+## Ricette
+
+- [Code coverage](docs/recipes/code-coverage.md)
+- [Test di Endpoint](docs/recipes/endpoint-testing.md)
+- [Quando usare `t.plan()`](docs/recipes/when-to-use-plan.md)
+- [Browser testing](docs/recipes/browser-testing.md)
+- [TypeScript](docs/recipes/typescript.md)
+
+
+## Supporto
+
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/ava)
+- [Gitter chat](https://gitter.im/sindresorhus/ava)
+- [Twitter](https://twitter.com/ava__js)
+
+
+## Guarda anche
+
+- [sublime-ava](https://github.com/sindresorhus/sublime-ava) - Frammenti di codice per test con AVA
+- [atom-ava](https://github.com/sindresorhus/atom-ava) - Frammenti di codice per test con AVA
+- [vscode-ava](https://github.com/samverschueren/vscode-ava) - Frammenti di codice per test con AVA
+- [eslint-plugin-ava](https://github.com/sindresorhus/eslint-plugin-ava) - Regole Lint per test AVA
+- [gulp-ava](https://github.com/sindresorhus/gulp-ava) - Esegui i test con gulp
+- [grunt-ava](https://github.com/sindresorhus/grunt-ava) - Esegui i test con grunt
+- [fly-ava](https://github.com/pine613/fly-ava) - Esegui i test con fly
+- [start-ava](https://github.com/start-runner/ava) - Esegui i test con start
+
+[Più moduli](https://github.com/sindresorhus/awesome-ava#packages)
+
+## Link
+
+- [Compra adesivi di AVA](https://www.stickermule.com/user/1070705604/stickers)
+- [Lista fantastica](https://github.com/sindresorhus/awesome-ava)
+
+
+## Creato da
```

Comment:
I've added a new section as in the English version, but this note doesn't disappear.